modules/javafx.graphics/src/main/java/com/sun/prism/impl/shape/DMarlinPrismUtils.java

```diff
@@ -105,6 +105,24 @@ private static DPathConsumer2D initStroker(
             final double b = tx.getMxy();
             final double c = tx.getMyx();
             final double d = tx.getMyy();
+            final double det = a * d - c * b;
+
+            if (Math.abs(det) <= (2.0d * Double.MIN_VALUE)) {
+                // this rendering engine takes one dimensional curves and turns
+                // them into 2D shapes by giving them width.
+                // However, if everything is to be passed through a singular
+                // transformation, these 2D shapes will be squashed down to 1D
+                // again so, nothing can be drawn.
+
+                // Every path needs an initial moveTo and a pathDone. If these
+                // are not there this causes a SIGSEGV in libawt.so (at the time
```

Comment:
Minor: Add space before so and S can be made capital.
Also wanted to check if libawt usage is correct here?